pytket/extensions/pyzx/pyzx_convert.py

```diff
@@ -229,7 +229,7 @@ def tk_to_pyzx_placed_circ(
         qubit_dict[q] = i
 
     for i, x in enumerate(pytket_arc.nodes):
-        arc_dict[x] = qubit_dict[q_map[x]]
+        arc_dict[x] = qubit_dict[q_map[x]]  # type: ignore
```

Comment:
I have not found a solution without an ignore, happy to take any suggestion for this